docs/contributor_guide/index.rst

```diff
@@ -47,4 +47,5 @@ All are welcome! Please reach out via
 `Discord <https://discord.gg/R22yCfgsRn>`_ and we will forward the
 invitation to you.
 We keep notes from each meeting in our
-`HackMD <https://hackmd.io/QcJ8WBAbQTq6iOgoi0JT4g>`_.
+`HackMD <https://hackmd.io/@Pu3O7yQTSFivjWn2cXCgFg/BkxG0KgrJl>`_.
+`Old HackMD <https://hackmd.io/QcJ8WBAbQTq6iOgoi0JT4g>
```

Comment:
```suggestion
`The notes from years 2020 and 2021 can be found here <https://hackmd.io/QcJ8WBAbQTq6iOgoi0JT4g>`_.
```

Reply:
Hi @milenaarneschedle! I added a little suggestion to improve the information about the older notes. You can commit it directly by clicking on the `commit suggestion` button right above this comment.
Could you please rebuild the documentation and take a screenshot of the updated change and how to looks?